caffe2/opt/fuse_conv_relu.cc

```diff
@@ -0,0 +1,88 @@
+#include "caffe2/opt/fuse_conv_relu.h"
+
+namespace caffe2 {
+namespace opt {
+
+using namespace nom;
+
+void FuseConvRelu(
```

Comment:
perhaps a lowercase function name `fuseConvRelu`?

Reply:
We are not using Google coding style?